contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -729,7 +731,7 @@ contract KeeperRegistry is
     address admin,
     uint96 balance,
     bytes memory checkData
-  ) internal {
+  ) internal whenNotPaused {
```

Comment:
I think this is something we're going to want to be able to do when paused. Can we remove this constraint?

Reply:
(≖_≖ ) .... ok moved